### PR TITLE
fighter build clarification + passenger seat change

### LINKED ISSUE #580: Can't enter a Sabre as a passenger without the proper access
## Reproduction:

1. Click on a Sabre with assistant level access

2. Access denied.

### PR DESCRIPTION
**Pull request by Karmic-Skink**
## About The Pull Request
This PR changes the method which passengers enter Sabre passenger seats to click-drag in order to bipass piloting access restrictions.
Also clarifies multitool build steps for fighters.

## Why It's Good For The Game
Shouldn't need access to be a passenger.
Fixes  #580 

## Changelog
:cl:
refactor: Changed the passenger entry method on Sabres from click to click-drag
tweak: Clarified fighter build state descriptions
/:cl:

### CODE REVIEW

File: nsv13/code/modules/overmap/fighters/fighters.dm [hunk 1]
```diff
@@ -458,6 +457,26 @@ You need to fire emag the fighter's IFF board. This makes it list as "ENEMY" on
 			escape_pod = EP
 			EP.flight_state = NO_IGNITION
 
+/obj/structure/overmap/fighter/MouseDrop_T(mob/living/M, mob/user) //Passengers get in via dragging themselves
```
Comment (Kmc2000) [inline nsv13/code/modules/overmap/fighters/fighters.dm:460]: Does this account for canopy being open? canopy should probably have to be open to hop in.

Comment (Karmic-Skink) [reply nsv13/code/modules/overmap/fighters/fighters.dm:460]: Doesn't account, assuming separate hatch.

Comment (Karmic-Skink) [reply nsv13/code/modules/overmap/fighters/fighters.dm:460]: Made it check for canopy being open.

File: nsv13/code/modules/overmap/fighters/fighters.dm [hunk 2]
```diff
@@ -458,6 +457,29 @@ You need to fire emag the fighter's IFF board. This makes it list as "ENEMY" on
 			escape_pod = EP
 			EP.flight_state = NO_IGNITION
 
+/obj/structure/overmap/fighter/MouseDrop_T(mob/living/M, mob/user) //Passengers get in via dragging themselves
+	.=..()
+	if(istype(M, /mob/living))
+		if(max_passengers <= 0)
+			return
```
Comment (Jalleo) [inline nsv13/code/modules/overmap/fighters/fighters.dm:464]: No feedback for no passenger slots?

Comment (Karmic-Skink) [reply nsv13/code/modules/overmap/fighters/fighters.dm:464]: Correct.

**Pull request merged**